v3/plugins/eclipse/cloud-shell/latest.txt

```diff
@@ -0,0 +1 @@
+nightly
```

Comment:
@amisevsk 
Is that version correct? 
Is it better to set something like 0.0.1?

Reply:
I guess correct version will be applied after release.

Reply:
I think setting `latest=nightly` makes sense here, it's a pre-release development build. The latest meta.yaml will be generated from the `nightly` meta.yaml during the registry build.